packages/uploadthing/src/internal/node-http/getBody.ts

```diff
@@ -0,0 +1,74 @@
+import type { IncomingMessage } from "http";
```

Comment:
very nit (and we should probably activate a lint rule to enforce it), but [i prefer `node:http`](https://github.com/pingdotgg/uploadthing/pull/280/files#diff-d029790c4e22075502109367243c609551aa27e5cb20615a35d004299d104571R2)

```suggestion
import type { IncomingMessage } from "node:http";
```

Reply:
Okay, will adjust